CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+- Ensure that discussions you have with other members stay within the framework of the Project. Apart from the situation when a module can resolve a problem encountered in the use of the PrestaShop solution or event sponsorship, spam or advertising messages are not welcome. 
+
+This list is not comprehensive. We expect the code to be followed in spirit as much as by the book. 
+
+
+## Unacceptable behaviours
+
+Certain behaviours are considered unacceptable and constitute a violation of this Code of Conduct, including:
+- Violation of rights, in particular, the right to privacy, intellectual property rights (such as copyright, trademark, patent law etc.), trade secrets, personal injury, etc. 
+- Rudeness and mockery, 
+- Insults, discriminations or personal attacks. This includes any language likely to offend or alienate people based on race, gender identity or expression, language, sexual orientation, disability, nationality, age, profession, physical appearance, religion, minority background,
+- Sexual or mental harassment, 
+- Verbal or physical violence, threats, intimidation, bullying, etc. This includes subtle belittlement message or unwelcoming language. No matter the intent, such behaviour can have a negative impact on others. 
+
+This Code of Conduct is not intended to be exhaustive or to detail all forms of unacceptable behaviours. It is not designed to rank the behaviours or to consider that some behaviours do not deserve to be punished. 
+
+If a member feels harassed or in an uncomfortable situation, they will be asked to report the situation according to the procedure described hereafter. 
+
+Every report will be examined. 
+
+
+## Reporting issues
+
+If you experience or witness misconduct, you may: 
+- Flag it thanks to the “report content” button on the contributory sites; 
+- Address it directly, and let the person know how this behaviour affects you or others;
+- If the situation is an online event, you may flag the content to the following address: [https://www.prestashop.com/en/abuse](https://www.prestashop.com/en/abuse);
+- If the situation happens during a physical event, you may address it directly to a member of the organization or a security officer. 
+
+Your report must include the followings: 
+- Your contact information, where you can be reached. Anonymous report cannot be treated; 
+- Your name and the name of the person or the people involved;
+- A description of what happened, and if you believe the situation is ongoing; don’t forget to provide the link of the webpage or the discussion; 
+- Any additional information that may be helpful.
+
+After filing a report, the most adapted person will handle the situation in order to check it. This person may ask you additional questions and will determine, with you, how to respond and handle the situation. 
```

Comment:
```suggestion
After the report has been filed, it will be forwarded to the appropriate person for verification and processing. This person may ask you additional questions and will determine, with you, how to respond and handle the situation. 
```